google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingImpl.java

```diff
@@ -195,6 +197,21 @@ LoggingOptions serviceOptions() {
     }
   }
 
+  private static class LogNamePageFetcher extends BasePageFetcher<String> {
+
+    private static final long serialVersionUID = 7156613795615829594L;
```

Comment:
what is this?

Reply:
What do you mean? A class for paged fetching has to be created for each type of request even if they return the same type of data.
All these classes have `serialVersionUID`, so the new class has it as well.